website/components/News.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import facepaint from 'facepaint';
+import ArrowRightIcon from '@leafygreen-ui/icon/dist/ArrowRight';
+import { css } from 'emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Overline, Subtitle, H2 } from '@leafygreen-ui/typography';
+import { spacing, breakpoints } from '@leafygreen-ui/tokens';
+
+const mq = facepaint(
+  Object.values(breakpoints).map(bp => `@media (min-width: ${bp}px)`),
+  { literal: true },
+);
+
+const newsContainer = css`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-bottom: ${spacing[4]}px;
+
+  ${mq({
+    marginTop: [`${spacing[4]}px`, 'unset'],
+    marginBottom: [`${spacing[4]}px`, 'unset'],
+    paddingLeft: [`${spacing[4]}px`, 'unset'],
+  })}
+`;
+
+const subtitleStyle = css`
+  cursor: pointer;
+  text-decoration: none;
+  font-weight: bolder;
+  display: inline-flex;
+  align-items: center;
+
+  &:hover {
+    & > svg {
+      opacity: 1;
+      transform: translate3d(3px, 0, 0px);
+    }
+  }
+`;
+
+const iconStyle = css`
+  transform: translate3d(-3px, 0, 0px);
+  transition: all 100ms ease-in;
+
+  ${mq({
+    opacity: [1, 1, 0, 0],
```

Comment:
I've added this attribute, but I actually think the best way to make this accessible is by making the role presentation and the alt text empty such that the icon is not announced to screenreaders. This requires an overhaul of our Icon component, though, because right now we can't overwrite the role="img" property

Reply:
>we can't overwrite the role="img" property

would wrapping the icon in an element with `aria-hidden="true"` achieve the same thing?

Reply:
yep, good call